lib/ns.c

```diff
@@ -413,12 +433,21 @@ void ns_init(void)
 
 	if (have_netns()) {
 		/* Install NS commands. */
-		install_node(&ns_node, ns_config_write);
 		install_element(CONFIG_NODE, &ns_netns_cmd);
```

Comment:
These should be not installed unless we've turned on namespaces to be used as vrf's right?